src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/WhereaboutsEnabledService.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+
+@Service
+class WhereaboutsEnabledService(
```

Comment:
More of a question, but does this need to be open?

Reply:
I'm not currently mocking it anywhere so technically no... but I'm open to it if you can think of a good reason?